### PR TITLE
[IOTDB-1077] [C++ Client] Add insertRecordsOfOneDevice() interface for C++ client

### PR DESCRIPTION
**Pull request by wshao08**
Add insertRecordsOfOneDevice() in C++ client to comply with Java Session interface

### CODE REVIEW

File: client-cpp/src/main/Session.cpp [hunk 1]
```diff
@@ -408,12 +417,52 @@ void Session::sortIndexByTimestamp(int* index, int64_t* timestamps, int length)
     }
 }
 
-vector<string> Session::sortList(vector<string>& valueList, int* index, int indexLength) {
-    vector<string> sortedValues(valueList.size());
-    for (int i = 0; i < indexLength; i++) {
-        sortedValues[i] = valueList[index[i]];
+void Session::putValuesIntoBuffer(vector<TSDataType::TSDataType>& types, vector<char*>& values, string buf) {
```
Comment (Alima777) [inline client-cpp/src/main/Session.cpp:420]:
```suggestion
void Session::putValuesIntoBuffer(vector<TSDataType::TSDataType>& types, vector<char*>& values, string& buf) {

```
Is reference here?

**Review by Alima777 — COMMENTED** [review]
Are there any tests about this??

File: client-cpp/src/main/Session.cpp (hunk 1 above)
Comment (wshao08) [reply client-cpp/src/main/Session.cpp]: Fixed

**Review by Alima777 — APPROVED** [review]
LGTM

**Pull request merged**